src/axom/slic/interface/slic.hpp

```diff
@@ -32,15 +32,30 @@ struct debug
 
 /*!
  * \brief Initializes the SLIC logging environment.
+ * \param [in] rootValue Enables selective logging macros based on root.
  */
-void initialize();
+void initialize(bool rootValue = true);
```

Comment:
Along the same theme:
```suggestion
void initialize(bool is_root = true);
```